sage/config.py

```diff
@@ -11,6 +11,9 @@
 
 from sage.reranker import RerankerProvider
 
+GEMINI_MAX_CHUNKS_PER_BATCH = 64
```

Comment:
Could you document where this number is coming from? Maybe I'm looking at the wrong place, but [this](https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings?authuser=0#get_text_embeddings_for_a_snippet_of_text) claims that it's 250 in us-central-1 and 5 in other regions.

Reply:
I couldn't find a value for this so just picked something that seemed reasonable